src/lib/versions.spec.js

```
@@ -0,0 +1,43 @@
+}
+
+describe('destroyFileVersion', () => {
+  it('deletes the version through the files route of a personal instance', async () => {
+    const { client, fetchJSON } = setup()
+
+    await destroyFileVersion({
+      client,
+      version: { id: 'file-id/2-fa3a3bec' }
+    })
+
+    expect(fetchJSON).toHaveBeenCalledWith('DELETE', '/files/file-id/2-fa3a3bec')
```

ESLint

[error] Replace `'DELETE',·'/files/file-id/2-fa3a3bec'` with `⏎······'DELETE',⏎······'/files/file-id/2-fa3a3bec'⏎····`

(prettier/prettier)